README.md

```diff
@@ -267,6 +267,9 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
 });
 ```
 
+**If you set the authorizer to authorizer:application** you need to create an `authorizers/application.js` inside your application (app/authorizers/applications.js). See the following section on implementing a custom authorizer.
+
```

Comment:
The README should focus on how things work in general rather than a specific case. Custom authorizers can be defined in `app/authorizers` while the specific name of these authorizers are up to the developer.